lib/shrinkwrapper.js

```diff
@@ -1,10 +1,12 @@
+/* eslint no-sync: 0 */
```

Comment:
Why is it so important that everything is done sync? I don't see a problem with it, but it seems unnecessary.